Sources/SwiftLanguageService/ParametersDocumentationExtractor.swift

```diff
@@ -0,0 +1,192 @@
+import Foundation
+import Markdown
+
+private struct ParametersDocumentationExtractor {
+  private var parameters = [String: String]()
```

Comment:
Nitpick, we tend to prefer this spelling.

```suggestion
  private var parameters: [String: String] = [:]
```

(same in a couple more locations)